docs/content/users/providers/lagoon.md

```diff
@@ -0,0 +1,18 @@
+# Amazee Lagoon Integration
+
+DDEV provides integration with the [Amazee Lagoon](https://lagoon.sh/), allowing Lagoon users to quickly download the files and database from a Lagoon project to the local DDEV project.
+
+## Amazee Lagoon Per-Project Configuration
+
+1. Check out the Lagoon project and configure it by running [`ddev config`](../usage/commands.md#config). You’ll want to run [`ddev start`](../usage/commands.md#start) and make sure the basic functionality is working.
+2. Configure an SSH key at on the [Amazee Cloud Dashboard](https://dashboard.amazeeio.cloud/settings).
+3. Add `LAGOON_PROJECT` and `LAGOON_ENVIRONMENT` variables to your project using `'web_environment'` in its YAML configuration or a `.ddev/.env` file. For example, run `ddev config --web-environment-add="LAGOON_PROJECT=<project-name> LAGOON_ENVIRONMENT=<environment-name>"`.
+4. Run `ddev auth ssh` to make your SSH key available in the project’s web container.
```

Comment:
Items 2 and 4 are directly related so I just moved them together. Also linking to Lagoon docs for SSH key instructions instead of amazee.io.

```suggestion
2. Add `LAGOON_PROJECT` and `LAGOON_ENVIRONMENT` variables to your project using `'web_environment'` in its YAML configuration or a `.ddev/.env` file. For example, run `ddev config --web-environment-add="LAGOON_PROJECT=<project-name> LAGOON_ENVIRONMENT=<environment-name>"`.
3. Configure an [SSH key](https://docs.lagoon.sh/using-lagoon-advanced/ssh/) for your Lagoon user.
4. Run `ddev auth ssh` to make your SSH key available in the project’s web container.
```